lib/coupon_code.rb

```diff
@@ -3,11 +3,11 @@
 require 'digest/sha1'
 
 module CouponCode
-  SYMBOL = '0123456789ABCDEFGHJKLMNPQRTUVWXY'
-  PARTS  = 3
-  LENGTH = 4
+  SYMBOL  = '0123456789ABCDEFGHJKLMNPQRTUVWXY'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.